plugin-template

```diff
@@ -61,7 +61,6 @@ DEFAULT_SETTINGS = {
 
 DEPRECATED_FILES = {
     "github": [
-        ".ci/scripts/cherrypick.sh",
```

Comment:
Maybe I should give this script a new name..

Reply:
Why, the template will replace the once deprecated one, right?

Reply:
Yes, it should.